electrumsv/gui/qt/network_dialog.py

```diff
@@ -1598,192 +1490,31 @@ def _enable_set_broadcast_service(self) -> None:
-            self._proxy_mode_combo.setCurrentText('SOCKS5')
-            self._proxy_host_edit.setText(self._tor_proxy[0])
-            self._proxy_port_edit.setText(str(self._tor_proxy[1]))
-            self._proxy_username_edit.setText("")
-            self._proxy_password_edit.setText("")
-            self._proxy_checkbox.setChecked(True)
-        else:
-            self._proxy_checkbox.setChecked(False)
-        self._check_disable_proxy(use_it)
-        self._set_proxy()
-
-    def _proxy_settings_changed(self) -> None:
-        self._tor_checkbox.setChecked(False)
-
-
-class TorDetector(QThread):
-    found_proxy = pyqtSignal(object)
-
-    def __init__(self) -> None:
-        QThread.__init__(self)
-
-    def run(self) -> None:
-        # Probable ports for Tor to listen at
-        ports = [9050, 9150]
-        for p in ports:
-            pair = ('localhost', p)
-            if TorDetector.is_tor_port(pair):
-                self.found_proxy.emit(pair)
-                return
-
-    @staticmethod
-    def is_tor_port(pair: Tuple[str, int]) -> bool:
-        try:
-            s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-            s.settimeout(0.1)
-            s.connect(pair)
-            # Tor responds uniquely to HTTP-like requests
-            s.send(b"GET\n")
-            if b"Tor is not an HTTP Proxy" in s.recv(1024):
-                return True
-        except socket.error:
-            pass
-        return False
-
-
 class NetworkTabsLayout(QVBoxLayout):
     def __init__(self, network: Network) -> None:
         super().__init__()
-        self._tor_proxy = None
         self._filling_in = False
         self._network = network
 
-        self._blockchain_tab = BlockchainTab(self, network)
+        # TODO(1.4.0) - replace with HeaderSV chain tips and local chain state this wallet follows
+        # self._blockchain_tab = BlockchainTab(self, network)
         self._servers_tab = ServersTab(self, network)
-        self._proxy_tab = ProxyTab(network)
 
         self._tabs = QTabWidget()
         self._tabs.setSizePolicy(QSizePolicy.Expanding, QSizePolicy.Expanding)
-        self._tabs.addTab(self._blockchain_tab, _('Blockchain Status'))
+
+        # TODO(1.4.0) - replace with HeaderSV chain tips and local chain state this wallet follows
+        # self._tabs.addTab(self._blockchain_tab, _('Blockchain Status'))
         self._tabs.addTab(self._servers_tab, _('Servers'))
-        self._tabs.addTab(self._proxy_tab, _('Proxy'))
 
         self.addWidget(self._tabs)
         self.setSizeConstraint(QVBoxLayout.SizeConstraint.SetFixedSize)
-        self._proxy_tab.set_tor_detector()
         self.last_values = None
 
-    def follow_server(self, server: SVServer, auto_connect: bool) -> None:
-        self._network.set_server(server, auto_connect)
-        # This updates the blockchain tab too.
-        self._servers_tab.update_servers()
+    # def follow_server(self, server: SVServer, auto_connect: bool) -> None:
```

Comment:
None of these orphaned comments should ever be checked in, they should be resolved or have a TODO(1.4.0) annotation.